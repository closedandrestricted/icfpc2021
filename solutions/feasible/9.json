{"vertices":[[133,57],[37,42],[141,46],[42,44],[116,54],[40,67],[114,46],[45,70],[35,52],[125,43],[39,53],[42,54],[45,51],[85,36],[63,62],[64,69],[59,136],[54,142],[26,23],[65,129],[59,42],[62,89],[31,12],[26,30],[23,31],[37,37],[72,147],[29,21],[76,137],[16,28],[41,41],[47,34],[27,31],[20,17],[20,22],[19,19],[35,39],[42,38],[21,28],[19,26],[23,32],[30,37],[38,35],[23,28],[22,36],[26,30],[25,25],[23,33],[22,30],[27,31],[33,35],[26,30],[16,18],[13,13],[30,36],[30,29],[23,36],[13,22],[13,62],[22,42],[24,43],[24,35],[25,29],[5,37],[47,46],[18,53],[16,41],[5,5],[18,32],[20,28],[9,12],[5,60],[15,52],[16,62],[8,29],[76,8],[34,28],[17,29],[8,35],[59,11],[72,13],[23,35],[10,39],[51,13],[27,43],[25,37],[56,13],[33,45],[29,42],[17,45],[57,21],[22,42],[15,41],[18,44],[66,30],[65,35]]}