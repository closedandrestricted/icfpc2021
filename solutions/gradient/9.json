{"vertices":[[8,43],[14,117],[8,29],[19,120],[13,60],[10,143],[19,55],[10,147],[18,128],[15,44],[18,131],[25,127],[23,131],[45,70],[36,103],[39,109],[70,129],[69,137],[97,60],[78,131],[57,95],[59,111],[82,78],[106,64],[104,65],[92,61],[70,155],[85,67],[73,145],[111,70],[73,77],[65,81],[91,59],[93,61],[99,47],[98,44],[76,72],[69,76],[84,61],[99,51],[108,54],[77,67],[75,74],[97,55],[106,50],[82,66],[99,53],[78,65],[78,66],[82,68],[80,75],[76,61],[144,144],[149,145],[79,64],[83,71],[77,54],[143,139],[93,11],[80,49],[83,50],[82,70],[86,67],[101,27],[110,108],[103,11],[82,43],[155,155],[85,66],[88,62],[149,149],[101,13],[85,35],[95,5],[100,64],[155,10],[123,59],[111,70],[116,74],[149,31],[153,19],[111,60],[118,71],[148,41],[104,67],[104,72],[153,40],[98,65],[98,70],[116,72],[154,48],[120,74],[120,75],[116,77],[155,60],[150,62]]}