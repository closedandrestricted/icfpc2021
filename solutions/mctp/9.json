{"vertices":[[11,90],[5,106],[5,82],[5,101],[5,101],[5,94],[5,109],[5,98],[5,112],[5,98],[5,109],[5,119],[5,115],[29,128],[13,98],[6,96],[5,99],[5,91],[5,92],[5,108],[36,102],[5,80],[13,83],[5,82],[5,84],[5,100],[5,104],[17,94],[5,93],[5,91],[19,103],[19,110],[20,87],[16,104],[5,85],[6,82],[18,98],[19,105],[20,96],[5,89],[5,81],[18,103],[19,110],[5,93],[5,85],[20,101],[5,90],[20,97],[19,106],[17,109],[19,115],[20,101],[5,85],[5,80],[20,114],[18,120],[17,107],[5,80],[5,88],[17,113],[15,113],[17,120],[17,125],[5,102],[5,117],[5,80],[16,121],[9,94],[17,126],[16,131],[5,89],[5,80],[5,125],[5,88],[17,145],[5,80],[5,138],[11,154],[6,128],[5,98],[5,87],[5,145],[5,131],[5,108],[5,152],[5,145],[5,103],[5,145],[5,138],[5,155],[5,97],[5,150],[5,150],[7,146],[5,85],[5,80]]}