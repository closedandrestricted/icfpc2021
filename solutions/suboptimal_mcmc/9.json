{"vertices":[[155,40],[76,123],[153,51],[80,125],[153,57],[110,129],[145,55],[115,132],[91,128],[148,63],[95,127],[93,121],[97,122],[121,74],[94,95],[88,99],[52,148],[47,154],[90,23],[59,142],[76,81],[68,103],[73,51],[94,14],[86,33],[95,40],[65,155],[71,40],[72,149],[90,24],[59,64],[68,64],[61,40],[64,44],[72,41],[72,44],[56,58],[62,61],[54,41],[70,38],[78,35],[53,52],[56,60],[67,37],[76,32],[53,45],[65,39],[49,46],[50,42],[52,46],[52,55],[45,44],[12,19],[6,19],[47,47],[47,55],[45,38],[19,20],[68,8],[47,32],[47,32],[47,54],[46,49],[63,18],[45,57],[75,5],[53,29],[5,5],[44,49],[40,48],[13,10],[61,14],[43,23],[66,6],[27,43],[19,52],[22,29],[16,45],[16,43],[34,57],[11,49],[21,54],[15,39],[34,48],[25,63],[23,58],[31,52],[23,57],[28,57],[14,61],[42,56],[10,57],[9,61],[5,60],[40,66],[45,70]]}