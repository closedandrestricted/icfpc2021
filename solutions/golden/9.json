{"vertices":[[9,42],[35,133],[8,28],[29,134],[13,59],[10,149],[19,54],[15,149],[19,132],[15,43],[18,136],[23,126],[21,130],[45,70],[36,103],[38,109],[79,127],[76,134],[90,46],[71,131],[55,91],[58,106],[85,77],[98,49],[96,52],[85,55],[71,152],[79,67],[65,144],[88,48],[71,78],[70,86],[88,65],[71,65],[104,53],[101,55],[77,77],[72,81],[79,68],[107,50],[102,56],[83,75],[77,82],[105,46],[105,54],[79,63],[102,45],[78,59],[84,63],[83,68],[77,75],[79,63],[137,147],[142,149],[87,71],[81,70],[78,56],[140,141],[93,11],[80,49],[76,48],[90,66],[88,71],[102,31],[108,109],[100,14],[79,42],[155,155],[87,62],[89,66],[148,149],[99,14],[85,35],[95,5],[102,62],[155,10],[125,61],[111,71],[121,48],[151,31],[154,19],[117,63],[118,49],[148,41],[113,70],[107,73],[152,40],[112,65],[107,67],[123,72],[145,50],[118,75],[120,75],[122,78],[154,60],[148,63]]}